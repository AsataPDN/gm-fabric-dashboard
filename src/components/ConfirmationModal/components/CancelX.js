import styled from "styled-components";

import { spacingScale } from "style/styleFunctions";

const CancelX = styled.span`
  cursor: pointer;
  height: ${spacingScale(3)};
  position: absolute;
  right: ${spacingScale(2)};
  top: ${spacingScale(2)};
  transition: all 0.3s ease;
  width: ${spacingScale(3)};

  &:hover,
  &:focus {
    transform: scale(1.25);
    transition: all 0.1s ease;
  }

  &:active {
    transform: scale(1.1);
    transition: all 0;
  }
`;

export default CancelX;
