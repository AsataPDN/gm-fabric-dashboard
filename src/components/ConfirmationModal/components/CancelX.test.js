import React from "react";
import { shallow } from "enzyme";
import CancelX from "./CancelX";

describe("CancelX", () => {
  it("matches snapshot", () => {
    const aCancelX = shallow(<CancelX />);
    expect(aCancelX).toMatchSnapshot();
  });
});
