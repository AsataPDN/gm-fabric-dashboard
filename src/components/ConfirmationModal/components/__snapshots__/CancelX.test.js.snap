// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CancelX matches snapshot 1`] = `
.c0 {
  cursor: pointer;
  height: 24px;
  position: absolute;
  right: 16px;
  top: 16px;
  -webkit-transition: all 0.3s ease;
  transition: all 0.3s ease;
  width: 24px;
}

.c0:hover,
.c0:focus {
  -webkit-transform: scale(1.25);
  -ms-transform: scale(1.25);
  transform: scale(1.25);
  -webkit-transition: all 0.1s ease;
  transition: all 0.1s ease;
}

.c0:active {
  -webkit-transform: scale(1.1);
  -ms-transform: scale(1.1);
  transform: scale(1.1);
  -webkit-transition: all 0;
  transition: all 0;
}

<span
  className="c0"
/>
`;
